test/e2e/web-map/mapAttributes.test.js

```diff
@@ -0,0 +1,27 @@
+import { test, expect, chromium } from '@playwright/test';
```

Comment:
Same comment as for the html file: name of mapAttributes is too generic, since you are only testing width and height, maybe make the file name mapWidthAndHeightAttributest.test.js